Bindings/Java/Matlab/Utilities/osimVec3FromArray.m

```diff
@@ -0,0 +1,34 @@
+% ----------------------------------------------------------------------- %
+% The OpenSim API is a toolkit for musculoskeletal modeling and           %
+% simulation. See http://opensim.stanford.edu and the NOTICE file         %
+% for more information. OpenSim is developed at Stanford University       %
+% and supported by the US National Institutes of Health (U54 GM072970,    %
+% R24 HD065690) and by DARPA through the Warrior Web program.             %
+%                                                                         %
+% Copyright (c) 2005-2016 Stanford University and the Authors             %
+% Author(s): James Dunne                                                  %
+%                                                                         %
+% Licensed under the Apache License, Version 2.0 (the "License");         %
+% you may not use this file except in compliance with the License.        %
+% You may obtain a copy of the License at                                 %
+% http://www.apache.org/licenses/LICENSE-2.0.                             %
+%                                                                         %
+% Unless required by applicable law or agreed to in writing, software     %
+% distributed under the License is distributed on an "AS IS" BASIS,       %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+% implied. See the License for the specific language governing            %
+% permissions and limitations under the License.                          %
+% ----------------------------------------------------------------------- %
+function vec = osimVec3FromArray(p)
+% import Java Libraries
```

Comment:
I think "import Java Libraries" will appear if someone types `help osimVec3FromArray` in MATLAB. Might be helpful to provide a short description (or nothing at all) instead.

Reply:
You're kinda correct. What actually comes up first is the license, since it is the first comment block in the script. I have added a help message at the top of each function. 

On a side note, this is why I think we should have the license on the bottom of the code, not at the top. Otherwise the order of the file has to (i) Function Description (ii) Licence (iii) Code. Seems weird.

Reply:
Ah, ok. Thanks for adding the descriptions.